src/communication/comm.js

```diff
@@ -0,0 +1,4 @@
+import { Bellhop } from 'bellhop-iframe';
+
+export const comm = new Bellhop();
```

Comment:
Could we rename this? I don't think `comm` clearly indicates intention. Something like `bellhopSingleton`. It's longer but clearer.

Reply:
Yes. I like that idea